testdata/traces/json/get-namespaces.har

```diff
@@ -0,0 +1 @@
+{"pageref":null,"startedDateTime":"","time":0,"request":{"method":"GET","url":"https://localhost:6445/api/v1/namespaces?limit=500","httpVersion":"unknown","cookies":[],"headers":[{"name":"Accept","value":"application/json"},{"name":"User-Agent","value":"kubectl/v1.13.0 (windows/amd64) kubernetes/ddf47ac"}],"queryString":[],"headersSize":-1,"bodySize":-1},"response":{"status":-1,"statusText":"","httpVersion":"unknown","cookies":[],"headers":[{"name":"Content-Type","value":"application/json"},{"name":"Content-Length","value":"1132"},{"name":"Date","value":"Thu, 21 Feb 2019 20:15:36 GMT"}],"content":{"size":-1,"mimeType":"","text":"eyJraW5kIjoiTmFtZXNwYWNlTGlzdCIsImFwaVZlcnNpb24iOiJ2MSIsIm1ldGFkYXRhIjp7InNlbGZMaW5rIjoiL2FwaS92MS9uYW1lc3BhY2VzIiwicmVzb3VyY2VWZXJzaW9uIjoiMTI2MDU4In0sIml0ZW1zIjpbeyJtZXRhZGF0YSI6eyJuYW1lIjoiZGVmYXVsdCIsInNlbGZMaW5rIjoiL2FwaS92MS9uYW1lc3BhY2VzL2RlZmF1bHQiLCJ1aWQiOiJiMjk0YTg5Zi0yYjQzLTExZTktYWYyOC0wMDE1NWQwMTFmMzciLCJyZXNvdXJjZVZlcnNpb24iOiI2IiwiY3JlYXRpb25UaW1lc3RhbXAiOiIyMDE5LTAyLTA4VDAxOjQ4OjU3WiJ9LCJzcGVjIjp7ImZpbmFsaXplcnMiOlsia3ViZXJuZXRlcyJdfSwic3RhdHVzIjp7InBoYXNlIjoiQWN0aXZlIn19LHsibWV0YWRhdGEiOnsibmFtZSI6ImRvY2tlciIsInNlbGZMaW5rIjoiL2FwaS92MS9uYW1lc3BhY2VzL2RvY2tlciIsInVpZCI6IjcwMGMwNTg5LTJiNDUtMTFlOS1hZjI4LTAwMTU1ZDAxMWYzNyIsInJlc291cmNlVmVyc2lvbiI6IjEyMzQiLCJjcmVhdGlvblRpbWVzdGFtcCI6IjIwMTktMDItMDhUMDI6MDE6MjRaIn0sInNwZWMiOnsiZmluYWxpemVycyI6WyJrdWJlcm5ldGVzIl19LCJzdGF0dXMiOnsicGhhc2UiOiJBY3RpdmUifX0seyJtZXRhZGF0YSI6eyJuYW1lIjoia3ViZS1wdWJsaWMiLCJzZWxmTGluayI6Ii9hcGkvdjEvbmFtZXNwYWNlcy9rdWJlLXB1YmxpYyIsInVpZCI6ImI1MGNjZmVlLTJiNDMtMTFlOS1hZjI4LTAwMTU1ZDAxMWYzNyIsInJlc291cmNlVmVyc2lvbiI6IjQ2IiwiY3JlYXRpb25UaW1lc3RhbXAiOiIyMDE5LTAyLTA4VDAxOjQ5OjAxWiJ9LCJzcGVjIjp7ImZpbmFsaXplcnMiOlsia3ViZXJuZXRlcyJdfSwic3RhdHVzIjp7InBoYXNlIjoiQWN0aXZlIn19LHsibWV0YWRhdGEiOnsibmFtZSI6Imt1YmUtc3lzdGVtIiwic2VsZkxpbmsiOiIvYXBpL3YxL25hbWVzcGFjZXMva3ViZS1zeXN0ZW0iLCJ1aWQiOiJiMzBlNGJlMS0yYjQzLTExZTktYWYyOC0wMDE1NWQwMTFmMzciLCJyZXNvdXJjZVZlcnNpb24iOiIxMiIsImNyZWF0aW9uVGltZXN0YW1wIjoiMjAxOS0wMi0wOFQwMTo0ODo1N1oifSwic3BlYyI6eyJmaW5hbGl6ZXJzIjpbImt1YmVybmV0ZXMiXX0sInN0YXR1cyI6eyJwaGFzZSI6IkFjdGl2ZSJ9fV19Cg==","encoding":"base64"},"redirectURL":"","headersSize":-1,"bodySize":-1},"cache":{},"timings":{"send":-1,"wait":-1,"receive":-1}}
```

Comment:
Ditto, can we reformat this for readability?
(This somewhat assumes we can convert the big base64 content block back into semi-readable text too - see above.  Encoded as json-in-json, it's always going to be a bit unreadable :disappointed: )